pom.xml

```diff
@@ -537,10 +537,6 @@
                     <groupId>io.netty</groupId>
                     <artifactId>netty-common</artifactId>
                 </exclusion>
-                <exclusion>
-                    <groupId>io.netty</groupId>
-                    <artifactId>netty-transport</artifactId>
```

Comment:
When I checked, there were indeed conflicts between minor versions of netty-transport. Restoring the exclusion seems to fix this.

Reply:
Thanks for checking on the netty JAR versions.

At some point, we should upgrade to the newer netty JARs, given the details on the FINP-9194 ticket. Since we seem to be transitively pulling in those JARs from the `secretsmanager` dependency, we should investigate upgrading that JAR in the near future, so that we'll automatically pull in the appropriate netty JAR versions.

I'll create a follow-up user story to look into upgrading to a newer `secretsmanager` version, to avoid scope creep for this PR's current changes.

Reply:
I've created KFSPTS-30649 to handle additional cleanup of the `secretsmanager` and `netty` dependencies. We'll address those dependencies as part of that ticket instead, so that we can keep the 01/25 patch moving forward.